README.md

```diff
@@ -1191,6 +1191,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [robobrowser](https://github.com/jmcarp/robobrowser) - A simple, Pythonic library for browsing the web without a standalone web browser.
 * [scrapy](https://scrapy.org/) - A fast high-level screen scraping and web crawling framework.
 * [portia](https://github.com/scrapinghub/portia) - Visual scraping for Scrapy.
+* [WebOOB](http://weboob.org/) - A framework for scraping sites with already existing site modules and ready-made client apps.
```

Comment:
Hello @hydrargyrum,
I see `Weboob` on the projects page, not `WebOOB`.

```suggestion
* [Weboob](http://weboob.org/) - A framework for scraping sites with already existing site modules and ready-made client apps.
```

Reply:
@Jamim : done

(actually both are ok, because "weboob" stands for "Web Out-Of-Browsers", but I did the change anyway)